app/domain/models/answerSheet.py

```diff
@@ -0,0 +1,33 @@
+from pydantic import BaseModel, Field
+
+class AnswerSheetModel(BaseModel):
+    adopter_id: str
+    form_id: str
+    score: int
+    answers: list[int] = []
+    
+    class Config:
+        schema_extra = {
+            "adopter_id": "653efb24188b172ef77d8acc",
+            "form_id": "653efee7eb19804a4a56fc86",
+            "score": 7,
+            "answers": [1,2]
+        }
```

Comment:
Esse deve ser o novo nome. `schema_extra` está deprecated

```suggestion
    class Config:
        json_schema_extra = {
            "adopter_id": "653efb24188b172ef77d8acc",
            "form_id": "653efee7eb19804a4a56fc86",
            "score": 7,
            "answers": [1,2]
        }
```